src/java/org/apache/cassandra/repair/RepairRunnable.java

```diff
@@ -314,6 +329,9 @@ private void notifyStarting()
 
     private NeighborsAndRanges getNeighborsAndRanges()
     {
+        if (options.isSkipNeighborValidation())
```

Comment:
what does neighbour validation mean?
"neighbour validation" is meaningless in itself, and the option is not documented.

Reply:
repair service is not a real C* node, it has no token metadata and it's not part of ring. how about changing it to `isOfflineService`?